src/FileIo.cpp

```diff
@@ -85,7 +85,9 @@ QString FileIo::pathFromUrl(QString const& _url)
 	if (url.scheme() == "file")
 	{
 		if (path.startsWith("/"))
-			path = path.right(path.length() - 1);
+			path = path.remove(0, 1);
+		if (path.startsWith("/"))
+			path = path.remove(0, 1);
```

Comment:
you are removing only one symbol -  "/" from the begining in first if, the second "/" could be only if there was one before